src/langcheck/augment/en/_rephrase.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+from typing import Optional
+
+import openai
+
+
+def rephrase(
+        instances: list[str] | str,
```

Comment:
Could you add a num_perturbations parameter so it's consistent with the other augmentations?